osgar/tools/lidarview.py

```diff
@@ -416,14 +416,15 @@ def _step(self, direction):
                 arr = deserialize(data)
                 assert len(arr) == 3
                 self.pose = (arr[0]/1000.0, arr[1]/1000.0, math.radians(arr[2]/100.0))
+            if stream_id in [self.pose2d_id, self.pose3d_id]:
```

Comment:
well, it is probably not correct - I would add there `elif stream_id in [self.pose2d_id, self.pose3d_id]:` and inside handle if pose2d else pose3d and them common crazy correction for g_rotation_offset_rad :(